test_conformance/c11_atomics/common.h

```diff
@@ -855,8 +955,9 @@ int CBasicTest<HostAtomicType, HostDataType>::ExecuteSingleTest(cl_device_id dev
     // Set up the kernel code
     programSource = PragmaHeader(deviceID)+ProgramHeader(numDestItems)+FunctionCode()+KernelCode(numDestItems);
     programLine = programSource.c_str();
-    if(create_single_kernel_helper_with_build_options(context, &program, &kernel, 1, &programLine, "test_atomic_kernel",
-      gOldAPI ? "" : "-cl-std=CL2.0"))
+    if (create_single_kernel_helper_with_build_options(
+            context, &program, &kernel, 1, &programLine, "test_atomic_kernel",
+            gOldAPI ? "" : nullptr))
```

Comment:
Sorry I didn't catch this earlier: I believe the updated helper logic creates programs with the latest OpenCL C version  unless another OpenCL C version is provided explicitly.  So, I think we should either pass `-cl-std=CL1.2" specifically in the `gOldAPI` case, or we can remove the `gOldAPI` case entirely and use the latest OpenCL C version in all cases.

This is low priority and it isn't breaking anything right now (I think) because the empty string `""` and NULL build options are treated the same, so we're using the latest OpenCL C version in all cases, even though it looks like there are two different behaviors.

Reply:
Good spot, thanks.

Unless we want to specifically make this test one of the "special" ones that explicitly specifies `-cl-std=CL1.2` then we can just use `nullptr` for all cases.